hosting/dependencies/runner.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+redis-server --requirepass $REDIS_PASSWORD > /dev/stdout 2>&1 &
+/bbcouch-runner.sh &
+/minio/minio server ${DATA_DIR}/minio --console-address ":9001" > /dev/stdout 2>&1 &
```

Comment:
It should not be a problem. This will happen in the container itself, docker will only expose it when required. And testcontainers will provide a random port